src/storage-controller/src/lib.rs

```diff
@@ -3146,8 +3160,6 @@ where
             _ => panic!("not a metrics history: {collection:?}"),
         };
 
-        let id = self.introspection_ids.lock().expect("poisoned")[&collection];
```

Comment:
We did this but had the `GlobalId` all along, must have been a vestige of an older state ... 🙈